apps/sms/views/shared/js/utils.js

```diff
@@ -711,7 +711,7 @@
        * flow is paused until yielded Promise is resolved, so that consumer gets
        * Promise result instead of Promise instance itself.
        * See https://www.promisejs.org/generators/ as the reference.
-       * @param {function*} generatorFunction Generator function that yields
+       * @param {*} generatorFunction Generator function that yields
```

Comment:
here we want to express we have a generator function as parameter (a function that, when called, returns an iterator).

Maybe `{function(...*): Iterator}\ ?